pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -69,6 +69,8 @@ const (
 	mxJobFailedReason = "MXJobFailed"
 	// mxJobRestarting is added in a mxjob when it is restarting.
 	mxJobRestartingReason = "MXJobRestarting"
+
+	FailedValidateJobReason = "MXJobValidationFailed"
```

Comment:
Minor:
Can you keep vars consistent with other frameworks ?We can think about keeping it common as well in the future

Reply:
Let me try it

Reply:
Done